docs/source/en/conceptual/pipeline_explained.mdx

```diff
@@ -0,0 +1,197 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Pipelines explained
```

Comment:
A different title? This guide explains one aspect of the pipeline regards to loading. Maybe "`DiffusionPipeline` explained"? ( `DiffusionPipeline` is the base model that's responsible for these tasks that you are talking about here)